app/package.json

```diff
@@ -56,9 +56,8 @@
     "winston-daily-rotate-file": "^1.4.5"
   },
   "devDependencies": {
-    "devtron": "^1.4.0",
```

Comment:
As far as I know, none of use (other than me) actually installed this plugin, so no need to keep it around

Reply:
I'd still keep this around if it's maintained upstream, [there are a bunch of nice benefits](https://electronjs.org/devtron).